README.md

```diff
@@ -209,3 +209,9 @@ it("can customize default config", async () => {
     }
 });
 ```
+
+# Release instructions
+Release automation has been setup according this [guide](https://michaelzanggl.com/articles/github-actions-cd-setup/).
+
+1. Create a Github release with version tag like `0.6.1`. 
+1. Check the new version exists on [npmjs.com/package/ts-retry-promise](https://www.npmjs.com/package/ts-retry-promise) and has `latest` tag. 
```

Comment:
This should probably be 1. and 2.

Reply:
I tend to use only 1.s as it makes inserting and reordering simpler. Will be render as 1.,2.,...

Reply:
Ah, didn't know that worked!